doc/advanced.rst

```diff
@@ -64,6 +64,27 @@ its reference
 it generated has the name ``sphx_glr_plot_0_sin_001.png``
 and its thumbnail is ``sphx_glr_plot_0_sin_thumb.png``
 
+You can also include part of a gallery script elsewhere in your documentation
+using the `literalinclude directive <https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude>`_, in order to limit code duplication::
```

Comment:
Is it possible to make this an intersphinx doc or ref link? It's more robust than a pure URL